lib/router/search.js

```diff
@@ -17,14 +17,24 @@ module.exports = (settings) => {
       .catch(next);
   });
 
-  app.get('/', (req, res, next) => {
+  app.param('index', (req, res, next, param) => {
+    if (!search.indexes.includes(param)) {
+      throw new NotFoundError();
+    }
+    next();
+  });
+
+  app.get('/:index', (req, res, next) => {
```

Comment:
I'd included the index in the query string rather than the path because I thought we might want to search multiple indexes at the same time (e.g. global search), but in that case I guess we'd probably just remove the index from the request entirely.

Reply:
I think searching multiple indexes in a single request is probably sufficiently far out of scope right now to disregard.